Sources/MapboxMaps/Foundation/Camera/BasicCameraAnimator.swift

```diff
@@ -49,20 +46,19 @@ public class BasicCameraAnimator: NSObject, CameraAnimator, CameraAnimatorInterf
     }
 
     // MARK: Initializer
-    internal init(delegate: CameraAnimatorDelegate,
-                  propertyAnimator: UIViewPropertyAnimator,
+    internal init(propertyAnimator: UIViewPropertyAnimator,
                   owner: AnimationOwner,
                   mapboxMap: BasicCameraAnimatorMapboxMap,
-                  cameraView: CameraView = CameraView()) {
-        self.delegate = delegate
```

Comment:
consider adding a precondition that the cameraView needs to be in the window.

Reply:
I've decided not to do this. It makes the unit tests worse, and provides limited value.